docs/client/api-box.html

```diff
@@ -1,16 +1,25 @@
 <template name="apiBoxTitle">
   <h3 id="{{id}}" class="api-title">
     <a class="name selflink" href="{{link}}">{{{name}}}</a>
-    {{#if locus}}
-      <span class="locus">{{locus}}</span>
-    {{/if}}
+    <div class="locus">
+        <span>
+        {{#if locus}}
+          {{locus}}
+        {{/if}}
+        </span>
+        {{#if filepath}}
+        <div class="src-code">
+            <a href="https://github.com/meteor/meteor/blob/devel/packages/{{filepath}}" target="_blank">{{filepath}}</a>,
```

Comment:
These should link to the `master` branch, which has the most recently uploaded copy of the docs (the devel docs sometimes have features that haven't been released yet, and therefore aren't in the deployed docs site)